sdk/monitor/monitor-opentelemetry-exporter/test/internal/statsbeat.test.ts

```diff
@@ -11,19 +11,29 @@ import nock from "nock";
 import { StatsbeatMetrics } from "../../src/export/statsbeat/statsbeatMetrics";
 // @ts-ignore Need to ignore this while we do not import types
 import sinon from "sinon";
-import { StatsbeatCounter } from "../../src/export/statsbeat/types";
+import {
+  StatsbeatCounter,
+  // StatsbeatFeature,
+  // StatsbeatInstrumentation,
+} from "../../src/export/statsbeat/types";
+import { LongIntervalStatsbeatMetrics } from "../../src/export/statsbeat/longIntervalStatsbeatMetrics";
 
 describe("#AzureMonitorStatsbeatExporter", () => {
+  // Represents REDIS and MONGODB instrumentations enabled
+  process.env.STATSBEAT_INSTRUMENTATIONS = "2,8";
```

Comment:
Instrumentation is a single number that represent a bitmap not a comma separated string

Reply:
This apply as well to the code that is parsing it and adding it in the Metric